lib/internal/error_serdes.js

```diff
@@ -48,13 +56,17 @@ function TryGetAllProperties(object, target = object) {
     if (getter && key !== '__proto__') {
       try {
         descriptor.value = FunctionPrototypeCall(getter, target);
+        delete descriptor.get;
+        delete descriptor.set;
```

Comment:
This will get called more times than before, but it can be refactored later, see next comment.